test/K6/src/data/testdata/at21testdata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "env": "at23",
```

Comment:
nitpick: The individual environment files still all reference `"env": "at23"`

Otherwise I like the idea of splitting the testdata in files pr. environment. As these might grow very big with time.

Reply:
Fixed the "env" references